MinPlatformPkg/Test/Library/TestPointLib/SmmTestPointCommunication.c

```diff
@@ -1,12 +1,33 @@
 /** @file
 
   Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) 2022, Microsoft Corporation. All rights reserved.<BR>
   SPDX-License-Identifier: BSD-2-Clause-Patent
 
 **/
 
+#include "MmTestPoint.h"
 #include "SmmTestPoint.h"
 
+/**
+  This function check if the buffer is valid per processor architecture and not overlap with SMRAM.
+
+  @param Buffer  The buffer start address to be checked.
+  @param Length  The buffer length to be checked.
+
+  @retval TRUE  This buffer is valid per processor architecture and not overlap with SMRAM.
+  @retval FALSE This buffer is not valid per processor architecture or overlap with SMRAM.
+**/
+BOOLEAN
+EFIAPI
+MmIsBufferOutsideMmValid (
```

Comment:
This is fine, but normally for this case, I created an abstracted function in the common local header. For example, we call the new interface as `MmIsBufferValid`, and for SMM instance, it will be `SmmIsBufferOutsideSmmValid` inside, and the Standalone MM will have `MmIsBufferOutsideMmValid`.

Reply:
Updated.